main.go

```diff
@@ -87,6 +88,11 @@ func StartCmd() cli.Command {
 				Usage: "Paths to a kubeconfig. Only required when it is out-of-cluster.",
 				Value: "",
 			},
+			cli.StringFlag{
+				Name:  FlagConfigMapName,
+				Usage: "Paths to a kubeconfig. Only required when it is out-of-cluster.",
```

Comment:
Wrong usage.

Reply:
ok, it is done.

Reply:
and i think i need to update the helm chart and after that you can review.

Reply:
it works for me using helm deployment now.